openedx/core/djangoapps/credentials/tests/factories.py

```diff
@@ -26,7 +26,7 @@ class Meta(object):
         model = dict
 
     credential_id = factory.Sequence(lambda n: n)
-    program_id = factory.Sequence(lambda n: n)
+    program_uuid = factory.Sequence(lambda n: n)
```

Comment:
Consider updating `factory_boy`. If everyone keeps passing the buck, it never gets done.